src/app/scenes/transaction/transaction.component.html

```diff
@@ -41,7 +41,7 @@ <h1>Explore your block</h1>
           <p>Processed at</p>
           <b>{{tx.created_at | date: 'yyyy/MM/dd HH:mm:ss'}} ({{tx.created_at | timeAgo }})</b>
         </li>
-        <li *ngIf="tx.contract_address">
+        <li *ngIf="tx.status && tx.contract_address">
```

Comment:
Is checking the status necessary/relevant? It seems like the literal `0x0000000000000000000000000000000000000000` is coming through, and a cleaner fix would be to change the backend to send null instead of a zero value `common.Address`.

Reply:
@bergauz ^